shell/client/accounts/login-buttons.js

```diff
@@ -65,8 +65,19 @@ function getActiveIdentityId(grains) {
 
 Template.accountButtons.helpers({
   displayName: function () {
+    if (Meteor.user() && !Meteor.user().loginIdentities) {
```

Comment:
Would be nice if we had an explicit `.ready()` on a subscription handle here.

I imagine this will show "Loading..." during the login-with-identity interstitial if you login as a non-login identity?